src/bundle/Resources/views/themes/admin/ui/form_fields/toggle_widget.html.twig

```diff
@@ -0,0 +1,52 @@
+{%- block toggle_widget -%}
+    {% set field_type = block_prefixes[1]|default('choice') %}
+    {% set main_class = 'ibexa-toggle' %}
+    {% set attr = attr|default({})|merge({
+        'class': (attr.class|default('') ~ ' ibexa-toggle__input ibexa-input ibexa-input--toggle')|trim,
```

Comment:
I wonder if `attr|default({})` would also work as default for `attr.class` ?

Reply:
Discussed in private